en/config_mc/mc_trajectory_tuning.md

```diff
@@ -12,37 +15,39 @@ The algorithm that generates those setpoints is called a "trajectory generator".
 ## Overview
 
 The input to the P/PID controller is a *desired setpoint* that the vehicle should attempt to track.
-[PID Tuning](../config_mc/pid_tuning_guide_multicopter.md) ("Lower level" tuning) aims to reduce the error between the desired setpoint and the estimate of the vehicle state.
-Poor P/PID Gains can lead to instability.
+[PID Tuning](../config_mc/pid_tuning_guide_multicopter.md) ("Lower level tuning") aims to reduce the error between the desired setpoint and the estimate of the vehicle state.
+
+The *desired setpoint* passed to the P/PID controller is itself calculated from a *demanded setpoint* based on a stick position (in RC modes) or from a mission command.
+The demanded setpoint can change very quickly (e.g. if a user moves stick from zero to maximum value as a "step").
+Vehicle flight characteristics are better if the corresponding desired setpoint changes as a "ramp".
 
-The *desired* setpoint passed to the P/PID controller is itself calculated from a *demanded* setpoint based on a stick position (in RC modes) or from a mission command.
-Setpoint value ("higher level") tuning is used to specify the mapping between the demanded setpoint and the desired setpoint.
-Poorly tuned setpoint values cannot result in instability, but may result in either very jerky or very unresponsive reactions to setpoint changes.
+*Setpoint value tuning* ("higher level tuning") is used to specify the mapping between the *demanded  and the *desired* setpoints - i.e. defining the "ramp" at which the desired setpoint follows the demanded setpoint.
 
-> **Tip** The demanded setpoint can change very quickly (e.g. if a user moves stick from zero to maximum value as a "step").
-  Vehicle flight characteristics are better if the corresponding desired setpoint changes as a "ramp".
+> **Tip** Poorly tuned [P/PID Gains](../config_mc/pid_tuning_guide_multicopter.md) can lead to instability.
+  Poorly tuned *setpoint values* cannot result in instability, but may result in either very jerky or very unresponsive reactions to setpoint changes.
 
-The setpoint-value tuning parameters can be split into two groups: tuning parameters for position mode and tuning parameters for mission mode.
-Some parameters will have an effect on both modes.
 
+<!-- 
```

Comment:
FYI, I commented this out because I don't think the definitions are used here or in the remaining document?